community-operators/percona-xtradb-cluster/percona-xtradb-cluster-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,238 @@
+              name: percona-xtradb-cluster-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: percona-xtradb-cluster-operator
+            spec:
+              containers:
+              - command:
+                - percona-xtradb-cluster-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: OPERATOR_NAME
+                  value: percona-xtradb-cluster-operator
+                image: percona/percona-xtradb-cluster-operator:1.0.0
+                imagePullPolicy: Always
+                name: percona-xtradb-cluster-operator
+                ports:
+                - containerPort: 60000
+                  name: metrics
+                resources: {}
+      permissions:
+      - rules:
+        - apiGroups:
+          - pxc.percona.com
+          resources:
+          - perconaxtradbclusters
+          - perconaxtradbclusters/status
+          - perconaxtradbclusterbackups
+          - perconaxtradbclusterbackups/status
+          - perconaxtradbclusterrestores
+          - perconaxtradbclusterrestores/status
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - pods/exec
+          - configmaps
+          - services
+          - persistentvolumeclaims
+          - secrets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - replicasets
+          - statefulsets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - batch
+          resources:
+          - jobs
+          - cronjobs
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - policy
+          resources:
+          - poddisruptionbudgets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - certmanager.k8s.io
+          resources:
+          - issuers
+          - certificates
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+          - deletecollection
+        serviceAccountName: default
+    strategy: deployment
+  maturity: alpha
+  provider:
+    name: Percona
+  version: 1.0.0
```

Comment:
This should also have a `replaces` field to denote that it replaces the v0.3.1 version.